.github/workflows/build_and_test.yml

```diff
@@ -31,11 +31,11 @@ jobs:
       pgupgrade_image_name: "citus/pgupgradetester"
       style_checker_image_name: "citus/stylechecker"
       style_checker_tools_version: "0.8.18"
-      image_suffix: "-v19b671f"
-      pg14_version: '{ "major": "14", "full": "14.10" }'
-      pg15_version: '{ "major": "15", "full": "15.5" }'
-      pg16_version: '{ "major": "16", "full": "16.1" }'
-      upgrade_pg_versions: "14.10-15.5-16.1"
+      image_suffix: "-dev-4f49b4a"
```

Comment:
Reminder that this needs to be replaced with a non-`dev` image before merging once https://github.com/citusdata/the-process/pull/155 has passed through CI